packages/web-components/src/components/ic-loading-indicator/ic-loading-indicator.stories.mdx

```diff
@@ -386,3 +386,28 @@ import readme from "./readme.md";
     }
   </Story>
 </Canvas>
+
+### Progress indicator for compact step
```

Comment:
The number is appearing above the progress indicator in this story (although it's fine in the actual stepper - appearing in the middle)